crates/dkg-core/src/node.rs

```diff
@@ -71,7 +71,9 @@ where
     type Next = Phase1<C>;
 
     fn run(self, board: &mut B, be_bad: bool) -> NodeResult<Self::Next> {
-        let (next, shares) = self.inner.shares();
+        // TODO: Should we remove `be_bad` and replace it with an RNG implementer,
```

Comment:
We should consider how to remove `be_bad` entirely. This was useful for an example and testing but not in a real usage scenario.

Reply:
Tracked inhttps://github.com/celo-org/celo-threshold-bls-rs/issues/43